src/helpers/files.js

```diff
@@ -22,14 +22,17 @@
 
 import { emit } from '@nextcloud/event-bus'
 import { Header } from '@nextcloud/files'
-import { imagePath } from '@nextcloud/router'
+import { imagePath, linkTo } from '@nextcloud/router'
 import { loadState } from '@nextcloud/initial-state'
 
 import { getSharingToken } from './token.js'
 import { openMimetypes } from './mime.js'
 import RichWorkspace from '../views/RichWorkspace.vue'
 import store from '../store/index.js'
 
+__webpack_nonce__ = window.btoa(OC.requestToken) // eslint-disable-line
```

Comment:
Usually we tend to have this in the actual entry point. I think we lack it in `src/init.js` which seems to be the only webpack entry point without it, so maybe we can also just move it there?